extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -317,6 +317,13 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
             };
           });
       sandbox.stub(impl, 'getAmpDoc', () => {return document;});
+      // Reproduced from noopMethods in ads/google/a4a/test/test-utils.js,
+      // to fix failures when this is run after 'gulp build', without a 'dist'.
+      sandbox.stub(impl, 'getPageLayoutBox', () => {
+        return {
+          top: 11, left: 12, right: 0, bottom: 0, width: 0, height: 0,
```

Comment:
Huh, okay. I'll look at them with our team later.

Also, I don't understand how any of this gulp dist stuff works and I think I should, but that shouldn't block this PR.